src/v8/v8.cc

```diff
@@ -441,7 +441,11 @@ bool V8::setWord(uint64_t pointer, Word word) {
   if (pointer + size > memory_->data_size()) {
     return false;
   }
+#if BYTE_ORDER == LITTLE_ENDIAN
```

Comment:
Where are those defines coming from? I'm not sure if this is the most portable way to detect endianess. I'd prefer using `__BYTE_ORDER__` and friends, since that's supported by both GCC and Clang:
```
#if defined(__BYTE_ORDER__) && defined(__ORDER_BIG_ENDIAN__) && __BYTE_ORDER__ == __ORDER_BIG_ENDIAN__
  uint32_t word32 = __builtin_bswap32(word.u32());
#else
  uint32_t word32 = word.u32();
#endif
```
Note the different order to fallback to little-endian in case of missing defines.